### PR TITLE
proxy server code and readme

### PR DESCRIPTION
**Pull request by beccaelenzil**
To be used with Weather Report: https://github.com/adagold/weather-report/tree/solution

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -11,3 +11,30 @@
...
+    loc_query = request.args.get("q")
+    if not loc_query:
+        return {"message": "must provide q parameter (location)"}
+
+    response = requests.get(
+        "https://us1.locationiq.com/v1/search.php",
+        params={"q": loc_query, "key": location_key, "format": "json"}
+    )
+
+    return jsonify(response.json())
```
Comment (kelsey-steven-ada) [inline app/routes.py:25]: If the response is already in JSON, do we need the `jsonify` call here?

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+    LOCATION_KEY="replace_with_your_api_key"
+
+    # OpenWeather API Key
+    WEATHER_KEY="replace_with_your_api_key"
+    ```
+
+1. Run the server
+    - `flask run`
+
+## Endpoints
```
Comment (kelsey-steven-ada) [inline README.md:29]: I know they are linked from the project, but would it be worth linking the API docs somewhere in this file for convenience?

File: app/routes.py (hunk 1 above)
Comment (beccaelenzil) [reply app/routes.py:25]: I just double checked. This is necessary since LocationIQ returns a list.

File: README.md (hunk 2 above)
Comment (beccaelenzil) [reply README.md:29]: Good call. I added this sentence to the intro "This server is a proxy for [LocationIQ](https://locationiq.com/docs) and [OpenWeather](https://openweathermap.org/api/one-call-api)"

**Pull request merged**